scripts/cfos/README.md

```diff
@@ -30,6 +30,10 @@
         2. upload flag - pushes generated load table tsv files to Terra workspace
             - requires workspace name and workspace project arguments
 
+	Output: 
+		- If errors occur, generates and populates validation_errors.csv with a list of all errors found. 
+		- If successful, generates a separate .tsv file for each table in the schema
```

Comment:
nice! to be clear, if successful, it will also upload those tsvs to Terra, right? I get why you didn't say that here since this is the outputs section but it could be helpful to mention in a parenthetical that those are the tsvs that also get ingested to the Terra data tables.